### PR TITLE
Influenzaandcovid19

### PR DESCRIPTION
**Pull request by kelcho-spense**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by kelcho-spense** [comment]
These are some of the some of Influenza and covid-19 flow pictures 
Screenshot 2022-10-06 142710
Screenshot 2022-10-06 142802
Screenshot 2022-10-06 142837
Screenshot 2022-10-06 142911
Screenshot 2022-10-06 142938
Screenshot 2022-10-06 142954

File: independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json [hunk 1]
```diff
@@ -0,0 +1,9216 @@
...
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v3/covid-19/states": {
+      "get": {
+        "responses": {
+          "default": {
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json:24]: @kelcho-spense Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

File: independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json [hunk 2]
```diff
@@ -0,0 +1,9216 @@
...
+            "type": "string",
+            "description": "",
+            "x-ms-visibility": "important",
+            "enum": [
+              "cases",
+              "todayCases",
+              "deaths",
+              "active"
+            ],
+            "x-ms-summary": ""
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json:118]: @kelcho-spense Empty property values can be removed until unless you have a specific reason to keep them.

File: independent-publisher-connectors/Influenza and Covid-19/apiProperties.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#f7f7f7",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Influenza and Covid-19/apiProperties.json:4]: @kelcho-spense The default iconBrandColor for any independent connector is `"#da3b01"`. Please follow the [templates](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/Independent%20Publisher).

File: independent-publisher-connectors/Influenza and Covid-19/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+    "capabilities": [],
+    "publisher": "Kevin Comba Gatimu , Denis Wachira Kathuri",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Influenza and Covid-19/apiProperties.json:6]: @kelcho-spense Please remove the space after `"Kevin Comba Gatimu"`.

File: independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json [hunk 5]
```diff
@@ -0,0 +1,9216 @@
...
+                      "description": "9/5/22"
+                    },
+                    "9/6/22": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "9/6/22"
+                    },
+                    "9/7/22": {
+                      "type": "integer",
+
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json:9106]: @kelcho-spense All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

**Review by sriyen-msft — DISMISSED** [review]

File: independent-publisher-connectors/Influenza and Covid-19/apiProperties.json (hunk 3 above)
Comment (kelcho-spense) [reply independent-publisher-connectors/Influenza and Covid-19/apiProperties.json:4]: I have rectified the error.

File: independent-publisher-connectors/Influenza and Covid-19/apiProperties.json (hunk 4 above)
Comment (kelcho-spense) [reply independent-publisher-connectors/Influenza and Covid-19/apiProperties.json:6]: I have rectified the error.

File: independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json (hunk 2 above)
Comment (kelcho-spense) [reply independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json:118]: I have rectified the error.

File: independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json (hunk 1 above)
Comment (kelcho-spense) [reply independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json:24]: I have rectified the error.

File: independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json (hunk 5 above)
Comment (kelcho-spense) [reply independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json:9106]: I have rectified the error.

**Comment by Amjed-Ayoub** [comment]
[[certify-connector]]

**Comment by Amjed-Ayoub** [comment]
Hello @kelcho-spense,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

**Comment by kelcho-spense** [comment]
Thank you very much, It was a learning experience I was happy to be part of. Looking forward to working with you in future

**Comment by Amjed-Ayoub** [comment]
Hello @kelcho-spense,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@kelcho-spense Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**